README.md

```diff
@@ -248,6 +248,92 @@ The generated css will be:
   }
 ```
 
+## Overriding styles
+
+When combining multiple aphrodite styles, you **must** merge all of your styles into a single call to `css()`, and **must not** combine the generated class names that aphrodite outputs (via string concatenation, `classnames`, etc.).
```

Comment:
Using the word `must` seems inflexible. I'd use "`should`" or "`it is recommended to`"